src/test/java/io/confluent/idesidecar/restapi/credentials/ApiKeyAndSecretTest.java

```diff
@@ -22,4 +24,17 @@ void shouldSerializeAndDeserialize() {
         ApiKeyAndSecret.class
     );
   }
+
+  @Test
+  void warpstreamLengthCredentialsShouldPassValidation() {
+    // 69 char key, 68 char secret from prior examples failing.
+    var key = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
+    var secret = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
```

Comment:
nit: Might be slightly easier to read.

```suggestion
    var key = "x".repeat(69);
    var secret = "x".repeat(68);
```